scipy-1.0/paper.tex

```diff
@@ -151,6 +151,34 @@ \subsection*{Numerical optimization}
 
 
 \subsection*{Statistical distributions}
+Version 1.0 of SciPy saw the addition of the energy distance algorithm in
+\texttt{scipy.stats.energy\_distance}. Energy distance is used in a varity of applications ranging
+from genetic analysis to multivariate independence. We also added the
+first Wasserstein distance algorithm to the library in
+\texttt{scipy.stats.wasserstein\_distance}. The Wasserstein distance is often
+used in color digital image analysis.
+
+We now also provide the cumulative distribution function of the multivariate
+normal distribution, \\\texttt{scipy.stats.multivariate\_normal}, via the newly
+associated \texttt{logcdf} and \texttt{cdf} methods.
+
+Other recent additions to \texttt{scipy.stats} include the Argus distribution
```

Comment:
Maybe all the distributions from this paragraph could be moved to the previous paragraph (so there is one paragraph that just lists all the new distributions)?